pantheon-bundle/frontend/src/app/versions.test.tsx

```diff
@@ -15,7 +15,7 @@ const props = {
     onGetProduct: (productValue) => anymatch,
     onGetVersion: (versionValue) => anymatch,
     productInfo: "Red Hat Enterprise Linux",
-    updateDate: (draftUpdateDate, releaseUpdateDate, releaseVersion, moduleUUID) => anymatch,
+    updateDate: (draftUpdateDate, releaseUpdateDate, releaseVersion, variantUUID) => anymatch,
     variant: "test",
     versionModulePath: "/modules/test/en_US/1"
```

Comment:
hmmm... I don't think this resource path is valid after the JCR changes.

Reply:
updated.